buildenv/jenkins/variables/defaults.yml

```diff
@@ -541,49 +551,11 @@ x86-64_linux_criu:
       11: ''
   build_env:
     cmd:
+      all: 'source /home/jenkins/set_gcc_10.3.0_env'
```

Comment:
I expect 8 should be set to `''` like in the x86-64_linux spec.

Reply:
I suppose 8 isn't supported atm since there are no build labels for it, but better to set it properly now just in case.

Reply:
Updated criu spec to for 8 too.